lib/helpers.js

```diff
@@ -62,7 +62,8 @@ exports.authorize = async ({
     expectedTarget, expectedRootCapability, expectedAction,
     inspectCapabilityChain: exports.inspectCapabilityChain,
     // TODO: support RsaSignature2018 and other suites?
-    suite: [new Ed25519Signature2018()]
+    suite: [new Ed25519Signature2018()],
+    allowHierarchicalDelegation: true,
```

Comment:
I *think* this is ok. This flag is always tempered by `expectedTarget` -- so whatever is listed there is what establishes the root of the hierarchy. We don't have any situations where we would accept `<edvs>` as a root expected target for say the `<edvs>/<edvId>/documents` endpoint. If we did, that would enable anyone who can create an EDV to be able to delegate zcaps to read any EDV. I think we're safe, just highlighting where we need to be careful with this: we have root of authority to create an EDV and root of authority to use a particular one -- and these shouldn't be mixed in the hierarchy (and currently are not).